vignettes/precompile.R

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+
+
+
+# Precompile vignettes
+
+
+library(knitr)
+
+
+# Convert *.orig to *.Rmd -------------------------------------------------
+
+knit("vignettes/bcdata.Rmd.orig", "vignettes/bcdata.Rmd")
+knit("vignettes/efficiently-query-spatial-data-in-the-bc-data-catalogue.Rmd.orig", "vignettes/efficiently-query-spatial-data-in-the-bc-data-catalogue.Rmd")
+
+
+# Move .png files into correct directory so they render -------------------
+
+images <- list.files(".", pattern = 'vignette-fig.*\\.png$')
+dest_images <- file.path("vignettes", images)
+
+success <- file.copy(images, dest_images, overwrite = TRUE)
+
+
+# Clean up if successful --------------------------------------------------
+
+if (!any(success)) {
```

Comment:
I think you want `all` here or this will always pass if at least one file copies successfully

```suggestion
if (!all(success)) {
```